packages/angular/cli/commands/deploy-impl.ts

```diff
@@ -19,7 +19,7 @@ favorite platform.
 For example:
   ng add @angular/fire
   ng add @azure/ng-deploy
-  ng add @zeit/ng-deploy
+  ng add angular-cli-ghpages
```

Comment:
I love `angular-cli-ghpages` and use it in my personal projects! It gets tricky to feature community modules without ignoring others. Our policy is to only recommend deployment builders that are maintained by the hosting vendor. In this case, if the package was under the `@github/*` scope we'd have been able to merge the PR. For now, I'd suggest just to remove `@zeit/ng-deploy`.

Reply:
I understand this, thanks for elaborating! 🙂 I'll remove the second commit.